analyses/cell-type-ewings/scripts/00-identify-ref-aucell.R

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env Rscript
+
+# This script is used to identify a reference threshold to use for classifying cells with AUCell
+# The reference SCE is run through AUCell using the marker gene list 
+# The auc threshold output by AUCell for this sample is returned as the output
+
+project_root <- here::here()
+renv::load(project_root)
+
+library(optparse)
+library(SingleCellExperiment)
```

Comment:
Minor, but I would might loading `SingleCellExperiment` until after the option parsing for faster failures (and maybe wrap the loading with `suppressPackageStartupMessages()`)